src/main/java/org/isf/dicom/service/DicomIoOperations.java

```diff
@@ -168,8 +169,12 @@ public boolean isCodePresent(Long code) throws OHServiceException {
 	 * @return <code>true</code> if the code is already in use, <code>false</code> otherwise
 	 * @throws OHServiceException
 	 */
-	public boolean isSeriePresent(String dicomSeriesNumber) throws OHServiceException {
-		List<FileDicom> result = repository.findAllWhereDicomSeriesNumber(dicomSeriesNumber);
-		return !result.isEmpty();
+	public boolean isSeriesPresent(String dicomSeriesNumber) throws OHServiceException
+	{
```

Comment:
Don't the braces usually go on the same line?

Reply:
yep! done. Thank you!